omegaup/validator/validatortest.py

```diff
@@ -51,7 +51,45 @@ def run(
         return result
 
 
-def main() -> None:
+class TestRunnerV2(unittest.TextTestRunner):
+    """A unittest.TestRunner that prints 1 on standard output on success.
+    If the associated .out file has contents, the test will consider
+    it to be the expected failure message for the case and will fail
+    with JE instead of WA. This is useful for invalid-inputs-enabled
+    CI runs."""
```

Comment:
```suggestion
    CI runs.
    """
```

por alguna razón Python decidió que la primer línea va junto a las comillas, pero la última no /shrug https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings